usage/jsgui/src/test/java/brooklyn/rest/jsgui/BrooklynJavascriptGuiLauncherTest.java

```diff
@@ -60,8 +61,15 @@ public void testJavascriptWithRest() throws Exception {
         checkUrlContains("/v1/catalog/entities", "Tomcat");
     }
 
-    protected void checkUrlContains(String path, String text) {
-        HttpTestUtils.assertContentContainsText(rootUrl()+path, text);
+    protected void checkUrlContains(final String path, final String text) {
+        //Server may return 403 until it loads completely, wait a bit
+        //until it stabilizes.
+        Asserts.succeedsEventually(new Runnable() {
```

Comment:
Can use `HttpTestUtils.assertContentEventuallyContainsText(rootUrl()+path, text)`.

I'd really like consistency for our assertXxxEventuallyYyy versus assertXxxYyyEventually. Personally, I much prefer the latter because we often also have an assertXxxYyy method - having the Eventually as a suffix keeps the methods together more. But that's an aside...